canvas_modules/common-canvas/locales/common-properties/locales/en.json

```diff
@@ -30,7 +30,7 @@
   "properties.support.message": "If the problem persists, contact support.",
   "properties.unexpected.message": "Oops, the pane didn't load.",
   "table.search.header": "View in tables",
-  "table.search.placeholder": "Find in column {column_name}",
```

Comment:
I think we could keep this.  Is there a reason it was changed?

Reply:
I thought we didn't want column name in the default search message. I will revert this change.